cmd/scollector/main.go

```diff
@@ -124,11 +125,12 @@ func main() {
 		check(collectors.AddProcessDotNetConfig(p))
 	}
 	for _, h := range conf.HTTPUnit {
+		freq := time.Second * time.Duration(h.Freq)
```

Comment:
We use `time.ParseDuration` in some places, but apparently not others. Is a number really better than a duration string if you have to look up the docs to know it is seconds?